Kitodo/src/main/java/org/kitodo/production/forms/ProcessForm.java

```diff
@@ -1182,4 +1182,11 @@ private int getProcessId(Object process) {
     public FilterMenu getFilterMenu() {
         return filterMenu;
     }
+
+    /**
+     * Rename media files of all selected processes.
+     */
+    public void renameMedia() {
+        ServiceManager.getFileService().renameMedia(getSelectedProcesses());
```

Comment:
I think this can also be all processes here. A limit should be built in here or implemented as a batch, e.g. via the task manager. What do you think?

Reply:
I added an optional parameter to `kitodo_config.properties` that can be used to limit the max number of processes allowed for media renmaing.